src/app/shared/components/template/components/task-card/task-card.component.ts

```diff
@@ -51,8 +126,13 @@ export class TmplTaskCardComponent extends TemplateBaseComponent implements OnIn
     this.title = getStringParamFromTemplateRow(this._row, "title", null);
     this.subtitle = getStringParamFromTemplateRow(this._row, "subtitle", null);
     this.image = getStringParamFromTemplateRow(this._row, "image", null);
+
+    // Use "variant" param to determine styles, including support for legacy "style" param
     this.style = getStringParamFromTemplateRow(this._row, "style", "landscape");
+    const variant = getVariantParamFromTemplateRow(this._row, "");
+    this.style = `${this.style} ${variant}`;
```

Comment:
nit(non-blocking)
I think if we treat variant the same as legacy style we might get ourselves a bit muddled down the road. I'd probably recommend keeping a separate `variant` or `style_variant` property and adding to specific css rules as required